src/components/UserAvatar/UserAvatar.types.ts

```diff
@@ -0,0 +1,15 @@
+import { UserAvatarSizes } from './UserAvatar.enums';
+
+export type Sizes = typeof UserAvatarSizes[keyof typeof UserAvatarSizes];
+
+export interface UserAvatarProps {
+  /**
+   * Used to render text elements inside the Avatar.
+   * It Will render the first two characters of the string always.
+   */
+  children: string;
```

Comment:
I'm thinking if we should use rather `label` property instead of `children` because if children are used people tend to customize the content and ask why we are restricting just to string

Reply:
@ajkl2533 I think you are right. Because even Pill and Badge are not using children and they could use it. So, I will change it to the label to be more aligned with overall